pkg/virt-launcher/monitor_test.go

```diff
@@ -105,7 +110,10 @@ var _ = Describe("VirtLauncher", func() {
 	}
 
 	BeforeEach(func() {
-		cmdlineMatchStr = uuid.New().String()
+		fakeUuid = uuid.New().String()
+		pidDir, err = ioutil.TempDir("", "kubevirt")
```

Comment:
I think you can use `os.MkdirTemp` and thus spare the use of `io/ioutil` package, since we use Go [1.17.x](https://github.com/enp0s3/kubevirt/blob/0c02044b3ae37d88bb4639b8e162c08f7f43dfd4/hack/builder/Dockerfile#L62)